### ISSUE TITLE
Owncloud mail app stuck on sending

### ISSUE THREAD

**Issue by tomaskruber** [owncloud/mail#1496]
### Steps to reproduce
1.Open email
2. Writing reply
3.Click on reply/send email

### Expected behaviour
Email sent, user should be informed that email was sent

### Actual behaviour
Email was sent, but button send is stuck on sending, and there is a running circle (loading)

### ownCloud Mail

**ownCloud Mail version:** (see ownCloud apps admin page)
0.4.4.
**Mailserver or service:** (e.g. Outlook, Yahoo, Gmail, Exchange,...)
Zoho

**Transport security - IMAP:** (None, SSL, TLS, STARTTLS) 
Incoming Server Name: imap.zoho.com
Port: 993
Require SSL: Yes  
Username : username@zoho.com

**Transport security - SMTP:** (None, SSL, TLS, STARTTLS)
Incoming Server Name: imappro.zoho.com
Port: 993
Require SSL: Yes  
Username : you@yourdomain.com 

**Number of accounts:**
1
**ownCloud Mail version build date:** (only if you are using a Nightly Build)

#### Server configuration
**Operating system**:
Linux 3.16.0-4-amd64 on x86_64, debian 8 
**Web server:**
Apache version 2.4.10	
**Database:**
mysql  Ver 14.14 Distrib 5.5.46, for debian-linux-gnu (x86_64) using readline 6.
**PHP version:**
PHP 5.6.14-0+deb8u1 (cli) (built: Oct  4 2015 16:13:10)
**ownCloud version:** (see ownCloud admin page)
8.2.5
**Updated from an older ownCloud or fresh install:**
Updated old
**List of activated apps:**
  - activity: 2.1.4
  - calendar: 0.8.2
  - contacts: 0.5.0.0
  - direct_menu: 0.7.1
  - external: 1.2
  - files: 1.2.1
  - files_external: 0.3.0
  - files_pdfviewer: 0.7.1
  - files_sharing: 0.7.0
  - files_trashbin: 0.7.0
  - files_versions: 1.1.0
  - firstrunwizard: 1.1
  - mail: 0.4.4
  - notifications: 0.1.0
  - provisioning_api: 0.3.0
  - shorten: 0.0.15
  - tasks: 0.8.1
  - templateeditor: 0.1
  - updater: 0.6
Disabled:
  - announcementcenter
  - calendar1
  - calendarplus
  - documents
  - encryption
  - files_texteditor
  - files_videoviewer
  - gallery
  - ojsxc
  - user_external
  - user_ldap
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**
        "asset-pipeline.enabled": true,
        "instanceid": "ocngddxvsngh",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "portal.educado.cz"
        ],
        "datadirectory": "\/var\/www\/owncloud\/data",
        "overwrite.cli.url": "http:\/\/192.168.88.2\/owncloud",
        "dbtype": "mysql",
        "version": "8.2.5.2",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "loglevel": 4,
        "theme": "",
        "maintenance": false,
        "appstore.experimental.enabled": true,
        "mail_smtpmode": "php",
        "log_rotate_size": 84857600

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

#### Client configuration
**Browser:**
Verze 50.0.2661.102 CHROME

**Operating system:**
WINDOWS 10

##### Web server error log
```



Insert your webserver log here
```

##### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here

```

##### Horde IMAP log (data/horde_imap.log)
```
Insert your horde IMAP log here, see https://github.com/owncloud/mail#debug-mode
```

##### Horde SMTP log (data/horde_smtp.log)
```

Insert your horde SMTP log here, see https://github.com/owncloud/mail#debug-mode
```

##### Browser log
```
Insert your browser log here, this could for example include:

https://portal.educado.cz/index.php/apps/mail/accounts/1/draft Failed to load resource: the server responded with a status of 500 ()
https://portal.educado.cz/index.php/apps/mail/accounts/1/send Failed to load resource: the server responded with a status of 500 ()
4c111dc….js:1072 Uncaught TypeError: Cannot read property 'status' of undefined(anonymous function) @ 4c111dc….js:1072c @ 4c111dc….js:1p.fireWith @ 4c111dc….js:1i.(anonymous function) @ 4c111dc….js:1u.error @ 4c111dc….js:1074c @ 4c111dc….js:1p.fireWith @ 4c111dc….js:1k @ 4c111dc….js:1r @ 4c111dc….js:1


a) The javascript console log
b) The network log 
c) ...
```

**Comment by tomaskruber** [comment]
OWncloud LOG:

{"reqId":"Td\/l3yoHFdBzcvEcZlL6","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:49:23+00:00"}
{"reqId":"Td\/l3yoHFdBzcvEcZlL6","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:49:23+00:00"}
{"reqId":"Td\/l3yoHFdBzcvEcZlL6","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:49:23+00:00"}
{"reqId":"Td\/l3yoHFdBzcvEcZlL6","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:49:23+00:00"}
{"reqId":"Td\/l3yoHFdBzcvEcZlL6","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:49:23+00:00"}
{"reqId":"Td\/l3yoHFdBzcvEcZlL6","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:49:23+00:00"}
{"reqId":"afRrZeL5+JN00ZqhGUjb","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:50:02+00:00"}
{"reqId":"afRrZeL5+JN00ZqhGUjb","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:50:02+00:00"}
{"reqId":"afRrZeL5+JN00ZqhGUjb","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:50:02+00:00"}
{"reqId":"afRrZeL5+JN00ZqhGUjb","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:50:02+00:00"}
{"reqId":"afRrZeL5+JN00ZqhGUjb","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:50:02+00:00"}
{"reqId":"afRrZeL5+JN00ZqhGUjb","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:50:02+00:00"}
{"reqId":"qd\/A09yw7F4sPisECJGE","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:51:12+00:00"}
{"reqId":"qd\/A09yw7F4sPisECJGE","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:51:12+00:00"}
{"reqId":"qd\/A09yw7F4sPisECJGE","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:51:12+00:00"}
{"reqId":"qd\/A09yw7F4sPisECJGE","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:51:12+00:00"}
{"reqId":"qd\/A09yw7F4sPisECJGE","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 1 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:51:12+00:00"}
{"reqId":"qd\/A09yw7F4sPisECJGE","remoteAddr":"141.101.97.237","app":"contacts","message":"OCA\\Contacts\\Contact::retrieve Error parsing carddata  for: 2 This parser only supports VCARD and VCALENDAR files","level":3,"time":"2016-05-17T09:51:12+00:00"}
{"reqId":"2BDs6dCSVSqxpavKuLnZ","remoteAddr":"141.101.97.237","app":"mail","message":"Saving draft failed: The object could not be created because it already exists.","level":3,"time":"2016-05-17T09:54:12+00:00"}
{"reqId":"5edoRr7gyb+RxJmflxX\/","remoteAddr":"141.101.97.238","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 423 \\\"EDUCADO (2)\\\/Certifik\\u00e1t\\\/ORDER #19335835 - Your Free EssentialSSL Site Seal.eml\\\" is locked\",\"Exception\":\"OC\\\\Connector\\\\Sabre\\\\Exception\\\\FileLocked\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(1087): OC\\\\Connector\\\\Sabre\\\\File->put(Resource id #35)\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/CorePlugin.php(501): Sabre\\\\DAV\\\\Server->updateFile('EDUCADO (2)\\\/Cer...', Resource id #35, NULL)\\n#2 [internal function]: Sabre\\\\DAV\\\\CorePlugin->httpPut(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#4 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(469): Sabre\\\\Event\\\\EventEmitter->emit('method:PUT', Array)\\n#5 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(254): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#6 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files\\\/appinfo\\\/remote.php(56): Sabre\\\\DAV\\\\Server->exec()\\n#7 \\\/var\\\/www\\\/owncloud\\\/remote.php(137): require_once('\\\/var\\\/www\\\/ownclo...')\\n#8 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/connector\\\/sabre\\\/file.php\",\"Line\":172}","level":4,"time":"2016-05-17T09:55:01+00:00"}
{"reqId":"DdEx7B1Be2hvojmigK0B","remoteAddr":"141.101.97.237","app":"mail","message":"Saving draft failed: The object could not be created because it already exists.","level":3,"time":"2016-05-17T10:00:43+00:00"}
{"reqId":"2\/3Nk8aMB3FB59nHIxMN","remoteAddr":"141.101.97.237","app":"mail","message":"Sending mail failed: The object could not be created because it already exists.","level":3,"time":"2016-05-17T10:00:52+00:00"}
{"reqId":"q1E2WzVUahmaHlHn5wid","remoteAddr":"141.101.97.238","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 423 \\\"EDUCADO (2)\\\/Certifik\\u00e1t\\\/ORDER #19335835 - Your Free EssentialSSL Site Seal.eml\\\" is locked\",\"Exception\":\"OC\\\\Connector\\\\Sabre\\\\Exception\\\\FileLocked\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(1087): OC\\\\Connector\\\\Sabre\\\\File->put(Resource id #35)\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/CorePlugin.php(501): Sabre\\\\DAV\\\\Server->updateFile('EDUCADO (2)\\\/Cer...', Resource id #35, NULL)\\n#2 [internal function]: Sabre\\\\DAV\\\\CorePlugin->httpPut(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#4 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(469): Sabre\\\\Event\\\\EventEmitter->emit('method:PUT', Array)\\n#5 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(254): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#6 \\\/var\\\/www\\\/owncloud\\\/apps\\\/files\\\/appinfo\\\/remote.php(56): Sabre\\\\DAV\\\\Server->exec()\\n#7 \\\/var\\\/www\\\/owncloud\\\/remote.php(137): require_once('\\\/var\\\/www\\\/ownclo...')\\n#8 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/connector\\\/sabre\\\/file.php\",\"Line\":172}","level":4,"time":"2016-05-17T10:05:02+00:00"}

**Comment by ChristophWurst** [comment]
Not tested, but I think this bug was fixed by d18c47d2ffc09e0041a0506359b5966bb660162e

**Comment by tomaskruber** [comment]
I changed the file, it did not work



Dne 17.05.2016 v 12:21 Christoph Wurst napsal(a):
>
> Not tested, but I think this bug was fixed by d18c47d 
> <https://github.com/owncloud/mail/commit/d18c47d2ffc09e0041a0506359b5966bb660162e>
>

-- 

*Tomáš Kruber*

+420 725 04 66 88

**Comment by ChristophWurst** [comment]
> I changed the file, it did not work

Did you enable debug mode?

**Comment by tomaskruber** [comment]
Thank you for support :-)

After enabling debug mode it shows server error after hitting send email

Dne 17.05.2016 v 13:51 Christoph Wurst napsal(a):
>
>     I changed the file, it did not work
>
> Did you enable debug mode?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/owncloud/mail/issues/1496#issuecomment-219695272>
>

-- 

*Tomáš Kruber*

+420 725 04 66 88

**Comment by ChristophWurst** [comment]
> After enabling debug mode it shows server error after hitting send email

Thanks for testing. This means the commit fixed that bug which will be released with 0.4.5.
